synfig-studio/src/gui/app.cpp

```diff
@@ -285,6 +285,7 @@ std::set< String > studio::App::brushes_path;
 String studio::App::image_editor_path;
 
 String studio::App::sequence_separator(".");
+String studio::App::number_of_cores("" + ('0'+g_get_num_processors()) );
```

Comment:
use standard c++ function : `std::to_string()` instead of ASCII tricks.
But why you store an integer number as text?!